build-system/tasks/prepend-global/index.js

```diff
@@ -198,7 +198,7 @@ async function getConfig(
  * @return {!Promise<string>}
  */
 async function getAmpConfigForFile(filename, options) {
-  const targets = options.minify ? MINIFIED_TARGETS : UNMINIFIED_TARGETS;
+  const targets = MINIFIED_TARGETS.concat(UNMINIFIED_TARGETS);
```

Comment:
Could you explain why this change is necessary? (The target names are supposed to be mutually exclusive for minified vs. unminified.)

Reply:
I don't need this update here. Can instead have `esbuildCompile` pass in the output name instead of src name.

IMO though, we can simplify the whole flow by not checking for `MINIFIED_TARGETS` at all, and only caring about UNMINIFIED